components/src/core/drawer/drawer-header/drawer-header.component.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { DrawerHeaderComponent } from './drawer-header.component';
+import { DrawerHeaderModule } from './drawer-header.module';
+import { Component } from '@angular/core';
+import { MatIconModule } from '@angular/material/icon';
+import { By } from '@angular/platform-browser';
+
+@Component({
+	template: `
```

Comment:
Can you run prettier on the components directory?

Reply:
ran prettier on component directory